src/game.js

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable class-methods-use-this */
+const Frame = require('./frame');
+const FinalFrame = require('./final_frame');
+
+class Game {
+  constructor() {
+    this.frames = [];
+  }
+
+  roll(one, two) {
+    this.rollParams(one, two);
+    this.newFrame(one, two);
+    if (this.frames.length > 2) {
+      this.bonusScan();
+    }
+  }
+
+  finalRoll(one, two) {
```

Comment:
Slightly better method names align with an action or verb describing what it's doing, eg. runFinalRoll, calculateFinalRoll, isFinalRoll, etc.